oioswift/proxy/controllers/obj.py

```diff
@@ -595,7 +599,8 @@ def _link_object(self, req):
         except exceptions.SourceReadError:
             req.client_disconnect = True
             self.app.logger.warning(
-                _('Client disconnected without sending last chunk'))
+                _('Client disconnected without sending last chunk') + (
```

Comment:
I don't know if a "link" operation can raise a `SourceReadError`

Reply:
Indeed, `SourceReadError` makes no sense in this context.